runtime/connectors/s3/s3.go

```diff
@@ -117,6 +118,15 @@ func (c connector) ConsumeAsFiles(ctx context.Context, env *connectors.Env, sour
 }
 
 func getAwsSessionConfig(ctx context.Context, conf *Config, bucket string) (*session.Session, error) {
+	// if S3CompatibleEndpoint is set then fs support s3 API
+	if len(conf.S3CompatibleEndpoint) > 0 {
+		return session.NewSession(&aws.Config{
+			Region:           aws.String("us-east-1"),
+			Endpoint:         &conf.S3CompatibleEndpoint,
+			S3ForcePathStyle: aws.Bool(true),
+		})
+	}
```

Comment:
Why does region need to be set when an endpoint is passed?

Also, what about credentials for S3 compatible endpoints? Should people use the AWS environment variables to override?

Reply:
The AWS SDK complains if region is not set. Cloudflare also allows user to set us-east-1 region for bwd compatibility reasons. 
Credentials work same like actual s3. I left some details here : https://github.com/rilldata/rill-developer/issues/1691#issuecomment-1415423446